src/wallet/test/wallet_tests.cpp

```diff
@@ -627,10 +627,17 @@ class ListCoinsTestingSetup : public TestChain100Setup
         BOOST_CHECK(wallet->CreateTransaction({recipient}, wtx, reservekey, fee, changePos, error, dummy));
         CValidationState state;
         BOOST_CHECK(wallet->CommitTransaction(wtx, reservekey, nullptr, state));
+        CMutableTransaction blocktx;
+        {
+        LOCK(wallet->cs_wallet);
+        auto it = wallet->mapWallet.find(wtx.GetHash());
+        BOOST_CHECK(it != wallet->mapWallet.end());
+        blocktx = CMutableTransaction(*it->second.tx);
+        }
+        CreateAndProcessBlock({CMutableTransaction(blocktx)}, GetScriptForRawPubKey(coinbaseKey.GetPubKey()));
```

Comment:
In commit "Require no cs_main lock for ProcessNewBlock/ActivateBestChain"

Maybe simplify this using mapWallet.at() instead of .find() and BOOST_CHECK.

Reply:
Done.